.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+    Thanks for making the effort to create a Pull Request!
+    Please read the comment in its entirety fist.
+    If you haven't already, please sign our Contributor License Agreement at
+    https://neo4j.com/developer/cla/
+    Commits from accounts that have not signed the CLA will fail the CI and
+    will not be reviewed. If you are a first-time contributor and have just
+    signed the CLA, please include a note about this in the PR comment as some
+    manual steps from our side are required.
+-->
+
+<!--
+    Title:
+    PRs targeting the current default branch (nightly driver) don't follow a
+    fixed naming scheme.
+    If your PR is a backport, please use the following naming scheme:
+    [targeted version] description
+    e.g.,
+    [4.4] Remove a banana from the code
```

Comment:
This seems to not match the workflow. On the other side this repo seems to use the `cherry-pick` label (which external contributers might not be able to set? not sure),  which is not a thing in the Python repo. So maybe you want to do something with that.